src/transdialog.cpp

```diff
@@ -702,32 +702,32 @@ bool mmTransDialog::ValidateData()
     //Checking account does not exceed limits
     if (m_new_trx || m_duplicate)
     {
-        bool abort_transaction = false;
-        double new_value = m_trx_data.TRANSAMOUNT;
-
-        if (m_trx_data.TRANSCODE == Model_Checking::all_type()[Model_Checking::WITHDRAWAL])
+        if ((m_trx_data.TRANSCODE == Model_Checking::all_type()[Model_Checking::WITHDRAWAL]) ||
```

Comment:
m_trx_data.TRANSCODE != Model_Checking::all_type()[Model_Checking::DEPOSIT]

Reply:
I figured that Model_Checking defines constants for the types; see my last commit.